### PR TITLE
Update credit.md with ORCID cv

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #<NUM>
Doesn't fix an issue but adds a resource to the ORCID part of the section

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)

* Add information on how to use your ORCID and an Rpackage to create a CV, saving you time!

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/reproducible-research/credit.md [hunk 1]
```diff
@@ -151,6 +151,7 @@ You can watch this short video for more information https://vimeo.com/97150912
 - All your outputs will remain linked to your unique identifier even if you change your name or your institute. This way, you only have to enter the information once.
 - To access or sign up to services that utilise ORCID, for example publisher requirements, funding management portals ([ResearchFish](ResearchFish)), the [CRIS (current research information system)](https://en.wikipedia.org/wiki/Current_research_information_system) system at your institute, like the [Zenodo](https://zenodo.org) repository.
 - You can add your ORCID to your CV/resume so that anyone can have a look at all your research outputs.
+- You can also use your ORCID to create a CV using, for example, the [ROpenSci package rorcid](https://discuss.ropensci.org/t/using-rorcid-to-generate-a-website-cv/1806).
```
Comment (malvikasharan) [inline book/website/reproducible-research/credit.md:154]:
```suggestion
- You can also use your ORCID profile to create a CV using [ROpenSci package rorcid](rorcid: https://github.com/ropensci/rorcid) as described [here](https://discuss.ropensci.org/t/using-rorcid-to-generate-a-website-cv/1806). 
```

**Review by malvikasharan — APPROVED** [review]
great idea!

**Comment by malvikasharan** [comment]
Thank you for the edits @EstherPlomp. It's ready to merge.

**Comment by EstherPlomp** [comment]
Thanks for the check and reminder! :D

**Pull request merged**